Automattic/wp-calypso
# Gutenberg: remove remaining usages of wordpress/nux package

#### Changes proposed in this Pull Request

Follow up for https://github.com/Automattic/wp-calypso/pull/28260.

We don't use core guided tour in Calypso so we don't need the DotTip component from this package. Since this is the last remaining usage, I'm removing the package dependency so we don't have to worry about it in future renovate updates.

Here is one example where it broke the editor during package updates https://github.com/Automattic/wp-calypso/pull/28000#discussion_r228925786.


#### Testing instructions

1. Navigate to http://calypso.localhost:3000/gutenberg/post
2. Smoke test the editor.
3. Verify that there are no other usages of `@wordpress/nux` package in Calypso's codebase.
